### PR TITLE
NEPT-1737: Update documentation to not use the user Poetry in tests.

### PR DESCRIPTION
**Pull request by voidtek**
### Description

Update documentation to not use the user Poetry in tests.

### Change log

- Changed: Update documentation to not use the user Poetry in tests.

### Commands

```sh
no command

```

### CODE REVIEW

File: profiles/common/modules/features/nexteuropa_dgt_connector/README.md [hunk 1]
```diff
@@ -105,68 +100,65 @@ contain:
...
+### DGT CONNECTOR PLUGIN SETTINGS:
+You should see **Main "poetry_service" variable is properly set.** if all
+the steps above were correctly followed. Otherwise, get back and check what
+you forgot!
+   
+### GENERAL SETTINGS [See DGT reference explanation]
+- *Counter*: **NEXT_EUROPA_COUNTER**
+- *Requester code*: **WEB** ([more information](#dgt-reference-number))
+- *Callback User / Callback Password*: drupal credential - in lowercase, limited to 15 characters and different to poetry credential. Example: **namesiteproduction/namesiteproduction**
+- *Poetry Username / Poetry Password*: Those credential are provided by DGT. Example **NE-SITENAME/NE-SITENAME**.
```
Comment (Fefaine) [inline profiles/common/modules/features/nexteuropa_dgt_connector/README.md]: In the email of Vincent Paoletti, after "Regarding your instructions" there is a highlighted sentence, bigger (not red but still) that says credentials are not provided by DGT
Are you trying to kill the poor guy? :p

File: profiles/common/modules/features/nexteuropa_dgt_connector/README.md [hunk 2]
```diff
@@ -105,68 +100,65 @@ contain:
...
+
+### DGT CONNECTOR PLUGIN SETTINGS:
+You should see **Main "poetry_service" variable is properly set.** if all
+the steps above were correctly followed. Otherwise, get back and check what
+you forgot!
+   
+### GENERAL SETTINGS [See DGT reference explanation]
+- *Counter*: **NEXT_EUROPA_COUNTER**
+- *Requester code*: **WEB** ([more information](#dgt-reference-number))
+- *Callback User / Callback Password*: drupal credential - in lowercase, limited to 15 characters and different to poetry credential. Example: **namesiteproduction/namesiteproduction**
```
Comment (Fefaine) [inline profiles/common/modules/features/nexteuropa_dgt_connector/README.md]: You don't give a list of examples that can be used. How is the site builder supposed to know how to test on playground if we don't say specifically in the readme?
namesiteproduction/namesiteproduction are not valide passwords

File: profiles/common/modules/features/nexteuropa_dgt_connector/README.md [hunk 3]
```diff
@@ -183,61 +175,58 @@ your production environment.
 
 # Usage
 ## How to request a translation
-- When you are browsing a node that is translatable, an additional tab
-'Translate' appears,
+- When you are browsing a node that is translatable, an additional tab **Translate** appears,
```
Comment (Fefaine) [inline profiles/common/modules/features/nexteuropa_dgt_connector/README.md]: line seems longer that the max limit

File: profiles/common/modules/features/nexteuropa_dgt_connector/README.md [hunk 4]
```diff
@@ -183,61 +175,58 @@ your production environment.
...
+can click **Change translator** to change it if needed,
+
 - You can change the languages you want to be translated,
-- Default values:
 
-> The values that show in 'Contact Usernames' and in 'Organization' are the
-default values you entered when [CEM configured your translator]
-(#configure-the-dgt-connector). Those values can be overridden on a page per
-page basis. To do this, just click 'Contact Usernames' or 'Organization' and
+- Default values: The values that show in **Contact Usernames** and in **Organization** are the
```
Comment (Fefaine) [inline profiles/common/modules/features/nexteuropa_dgt_connector/README.md]: line seems to long that the max limit

File: profiles/common/modules/features/nexteuropa_dgt_connector/README.md [hunk 5]
```diff
@@ -105,68 +100,65 @@ contain:
...
-   Typical example is when the site is configured for using 'Portuguese from 
-   Portugal' (code pt-pt) that should be mapped to 'pt'.
+> Configuration > Regional and Language > Translation management Translator
+
+and edits 'DGT Connector', or go to:
+
+> admin/config/regional/tmgmt_translator/manage/poetry 
+
+### TRANSLATOR SETTINGS
+- *Auto accept finished translations*: Check this if the site owner wants to review a translation before
```
Comment (Fefaine) [inline profiles/common/modules/features/nexteuropa_dgt_connector/README.md]: line seems longer that the max limit

File: profiles/common/modules/features/nexteuropa_dgt_connector/README.md (hunk 2 above)
Comment (voidtek) [reply profiles/common/modules/features/nexteuropa_dgt_connector/README.md]: For the Callback User/ Callback Password, we can use any user/password. It is the API identifier on the drupal side. If you want, you can use totoismyhero / blablaisthepassword, it works.
The only detail we need to be careful about is that these credentials must be unique for each instance. Do not use the same credentials for playground / production / localdev.

File: profiles/common/modules/features/nexteuropa_dgt_connector/README.md (hunk 1 above)
Comment (voidtek) [reply profiles/common/modules/features/nexteuropa_dgt_connector/README.md]: I removed this information and put more information according the last communication of Vincent.

**Pull request closed without merging**